packages/datatrak-web/src/features/Survey/utils.ts

```diff
@@ -26,51 +15,42 @@ export const getSurveyScreenNumber = (screens, screen) => {
   if (!screen) return null;
   const { surveyScreenComponents, id } = screen;
   const nonInstructionScreens =
-    screens?.filter(screen =>
-      screen.surveyScreenComponents.some(component => component.type !== QuestionType.Instruction),
+    screens?.filter((screen) =>
+      screen.surveyScreenComponents.some(
+        (component) => component.type !== QuestionType.Instruction
+      )
     ) ?? [];
 
   const screenNumber = surveyScreenComponents.some(
-    component => component.type !== QuestionType.Instruction,
+    (component) => component.type !== QuestionType.Instruction
   )
-    ? nonInstructionScreens.findIndex(nonInstructionScreen => nonInstructionScreen.id === id) + 1
+    ? nonInstructionScreens.findIndex(
+        (nonInstructionScreen) => nonInstructionScreen.id === id
+      ) + 1
     : null;
 
   return screenNumber;
 };
 
-export const formatSurveyScreenQuestions = (
-  questions: SurveyScreenComponent[],
-  screenNumber: number | string,
-) => {
-  const nonReadOnlyQuestions = questions.filter(
-    question => !READ_ONLY_QUESTION_TYPES.includes(question?.type!),
-  );
-
-  return questions.map(question => {
-    const questionNumber = nonReadOnlyQuestions.findIndex(
-      nonInstructionQuestion => question.questionId === nonInstructionQuestion.questionId,
-    );
-    if (questionNumber === -1) return question;
-    return {
-      ...question,
-      questionNumber: `${screenNumber}${convertNumberToLetter(questionNumber)}.`,
-    };
-  });
-};
-
 export const getAllSurveyComponents = (surveyScreens?: SurveyScreen[]) => {
-  return surveyScreens?.map(({ surveyScreenComponents }) => surveyScreenComponents)?.flat() ?? [];
+  return (
+    surveyScreens
+      ?.map(({ surveyScreenComponents }) => surveyScreenComponents)
+      ?.flat() ?? []
+  );
```

Comment:
This might be a good place for `flatMap()`?

```tsx
  return (
    surveyScreens?.flatMap(({ surveyScreenComponents }) => surveyScreenComponents) ?? []
  );
```

(Not a real suggestion because my format’s probably wrong)